test/interpreter/function-hlookup.spec.ts

```diff
@@ -266,4 +266,33 @@ describe('Function HLOOKUP', () => {
     ], {binarySearchThreshold: 1})
     expect(engine.getCellValue(adr('A1'))).toEqual('b')
   })
+
+  it('works for strings, is not case sensitive', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['a', 'b', 'c', 'A', 'B'],
+      [1, 2, 3, 4, 5],
+      ['=HLOOKUP("A", A1:E2, 2, FALSE())']
+    ])
```

Comment:
Should we add `caseSensitive: false` to each test without it? In case the default changes in the future we will have those tests sealed
```suggestion
    ], , {caseSensitive: false})
```